ws.py

```diff
@@ -80,12 +79,21 @@
 
 if "ChatExchangeU" in os.environ:
     username = os.environ["ChatExchangeU"]
+elif GlobalVars.chatexchange_u:
+    log('info', "ChatExchange username loaded from config")
+    username = GlobalVars.chatexchange_u
 else:
-    username = input("Username: ")
+    log('error', "No ChatExchange username provided. Set it in config or provide it via environment variable")
+    os._exit(6)
+
 if "ChatExchangeP" in os.environ:
     password = os.environ["ChatExchangeP"]
+elif GlobalVars.chatexchange_p:
+    log('info', "ChatExchange password loaded from config")
```

Comment:
Can this be debug not info? It's only really going to be relevant if you're debugging an issue - in most cases, you're going to know where it was loaded from.